tailwind.config.js

```diff
@@ -90,11 +90,16 @@ module.exports = {
         }
       }
     },
+    maxHeight: {
+      '32': '8rem'
+    }
+  },
+  variants: {
+    padding: ['last']
```

Comment:
We need to specify which properties we can use `last:` on, so here I am saying we can now set it for `padding`.

Tailwind say they don't default everything to have it because it would increase the bundle size considerably.